core/src/sql/kind.rs

```diff
@@ -30,6 +36,7 @@ pub enum Kind {
 	Set(Box<Kind>, Option<u64>),
 	Array(Box<Kind>, Option<u64>),
 	Function(Option<Vec<Kind>>, Option<Box<Kind>>),
+	Literal(Literal),
```

Comment:
Doesn't that need a new revision?

Reply:
Not needed here, with revision you can safely add new enum entries at the end without bumping the revision version